mix.lock

```diff
@@ -1,5 +1,4 @@
 %{"bbmustache": {:hex, :bbmustache, "1.0.4", "7ba94f971c5afd7b6617918a4bb74705e36cab36eb84b19b6a1b7ee06427aa38", [:rebar], []},
```

Comment:
Its a subdependency of something else. But thats neat will just have to add it to applications: [] then